control/latlon_muxer/CMakeLists.txt

```diff
@@ -1,33 +1,41 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(latlon_muxer)
 
-add_compile_options(-std=c++14)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
+endif()
 
-find_package(
-  catkin REQUIRED COMPONENTS
-    roscpp
-    autoware_control_msgs
-)
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(autoware_control_msgs REQUIRED)
```

Comment:
I find it easier (following the DRY principle) to do
```
find_package(ament_cmake_auto REQUIRED)
ament_auto_find_build_dependencies()
```
then you neither have to set `LATLON_MUXER_DEPENDENCIES` nor call `ament_target_dependencies` nor `target_include_directories` below. See e.g. my PR https://github.com/tier4/Pilot.Auto/pull/7/files#diff-a6afdb956fcb25615e51250834552921

Reply:
Yeah, I prefer ament_auto too, but I thought it was out of scope for the porting. I'm happy to switch to it, though. @mitsudome-r what's the plan regarding ament_auto?

Reply:
Let's use ament_auto. I think it would make management of dependent package much easier.